src/main/java/com/conveyal/datatools/editor/models/transit/TripPattern.java

```diff
@@ -47,7 +45,7 @@
 
     public TripDirection patternDirection;
 
-    public List<TripPatternStop> patternStops = new ArrayList<TripPatternStop>();
+    public List<TripPatternStop> patternStops = new ArrayList<>();
 
     // give the UI a little information about the content of this trip pattern
     public transient Long numberOfTrips;
```

Comment:
Ah, was just mimicking the same field type in ServiceCalendar.  Definitely doesn't need to be a long.